### ISSUE TITLE
MMS test required for iso elastic wave case

### ISSUE THREAD

**Issue by ggorman** [opesci/opesci-fd#3]
Follow recipe given in http://prod.sandia.gov/techlib/access-control.cgi/2000/001444.pdf to create MMS verification test for elastic wave equation example.

**Comment by ggorman** [comment]
Completed by @tj-sun

**Comment by ctjacobs** [comment]
I don't think this is true - the eigenwave test is not really the same as an MMS test. As the source term implementation needs re-doing anyway (issue #6), I think we should keep this open and create a proper MMS test to make sure the source term implementation is working properly.

**Comment by tj-sun** [comment]
agreed
